ginkgo/notifications.go

```diff
@@ -59,3 +61,27 @@ func (n *Notifier) SendNotification(title string, subtitle string) {
 		exec.Command("terminal-notifier", args...).Run()
 	}
 }
+
+func (n *Notifier) RunCommand(suite testsuite.TestSuite, suitePassed bool) {
```

Comment:
Not sure if this is the right place for this, but it was the only place I saw with access to the command line args from the suite completion.